cmd/rwx/image/build.go

```diff
@@ -0,0 +1,63 @@
+package image
+
+import (
+	"time"
+
+	"github.com/rwx-cloud/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildInitParameters []string
+	buildRwxDirectory   string
+	buildMintFilePath   string
+	buildNoCache        bool
+	buildTargetTaskKey  string
+	buildTags           []string
+	buildTimeout        time.Duration
+
+	BuildCmd *cobra.Command
+)
+
+func InitBuild(requireAccessToken func() error, parseInitParameters func([]string) (map[string]string, error), service cli.Service) {
+	BuildCmd = &cobra.Command{
+		Args: cobra.MaximumNArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return requireAccessToken()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
+				buildMintFilePath = args[0]
+			}
+
+			initParams, err := parseInitParameters(buildInitParameters)
+			if err != nil {
+				return err
+			}
+
+			config := cli.BuildImageConfig{
+				InitParameters: initParams,
+				RwxDirectory:   buildRwxDirectory,
+				MintFilePath:   buildMintFilePath,
+				NoCache:        buildNoCache,
+				TargetTaskKey:  buildTargetTaskKey,
+				Tags:           buildTags,
+				Timeout:        buildTimeout,
+			}
+
+			return service.BuildImage(config)
+		},
+		Short: "Launch a targeted rwx run and pull its result as an OCI image",
+		Use:   "build <file> --target <task-key> [flags]",
+	}
+
+	BuildCmd.Flags().StringArrayVar(&buildInitParameters, "init", []string{}, "initialization parameters for the run, available in the `init` context. Can be specified multiple times")
+	BuildCmd.Flags().StringVarP(&buildMintFilePath, "file", "f", "", "an RWX config file to use for sourcing task definitions (required)")
+	BuildCmd.Flags().StringVarP(&buildRwxDirectory, "dir", "d", "", "the directory your RWX configuration files are located in, typically `.rwx`. By default, the CLI traverses up until it finds a `.rwx` directory.")
```

Comment:
dir is a positional arg now? why is that?

Reply:
It's not although looking at this, I should just reuse `addRwxDirFlag`

But the `P` in the method name is an indication that the short flag `-d` is also accepted: 

https://github.com/spf13/pflag/blob/6fcfbc9910e1af538fde31db820be7d1bec231e4/string.go#L51

Reply:
I'm going to leave it as is in this PR and do a bit of a refactor on how the whole CLI is constructed in another PR.